src/components/organisms/header/header.tsx

```diff
@@ -7,7 +7,7 @@ const Header = () => {
   const navList = useMainNav();
 
   return (
-    <header className="flex bg-white h-24 w-full sticky top-0 justify-between items-center border-b px-4 py-2">
+    <header className="flex z-[999] bg-white h-24 w-full sticky top-0 justify-between items-center border-b px-4 py-2">
```

Comment:
```suggestion
    <header className="flex bg-white h-24 w-full sticky top-0 justify-between items-center border-b px-4 py-2">
```

Please remove this. You were right to add the z-index here as images will hover over the header but it is fixed in another PR and this will simply cause a conflict. No need to worry about it here, it'll be fixed when the other PR is merged. Thank you!